apps/greeter/lib/greeter.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Greeter do
+  use Application
+
+  # See http://elixir-lang.org/docs/stable/elixir/Application.html
+  # for more information on OTP Applications
+  def start(_type, _args) do
+    # import Supervisor.Spec
+
+    # Define workers and child supervisors to be supervised
+    children = [
+      # Start the endpoint when the application starts
+      # supervisor(Greeter.Endpoint, []),
```

Comment:
We are not loading the endpoint so we can `mix phoenix.server` in the root of the umbrella project